perun-base/src/main/resources/perun-roles.yml

```diff
@@ -920,42 +934,25 @@ perun_policies:
       - FACILITYADMIN: Facility
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Facility
 
   addHosts_Facility_List<String>_policy:
     policy_roles:
       - FACILITYADMIN: Facility
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Facility
 
   removeHosts_List<Host>_Facility_policy:
     policy_roles:
       - FACILITYADMIN: Facility
     include_policies:
       - default_policy
-
-  addAdmin_Facility_User_policy:
-    policy_roles:
-      - FACILITYADMIN: Facility
-    include_policies:
-      - default_policy
-
-  addAdmin_Facility_Group_policy:
-    policy_roles:
```

Comment:
Why delete all of this? I see that these policies aren't used in the Entry layers of the methods, however shouldn't they still require MFA? Are we somehow going to implement MFA into setRole() or am I missing something?

Reply:
good point, we'll want to have setRole() methods covered. that method is using `authorizedToManageRole` check, which throws PrivilegeException, so I imagine we could create public method in Utils like isMfaCritical and send the complementary object to it, then decide if MFA requirements are met etc. - so not using perun-roles for this use case at all, but handle it separately. if you'd agree with this solution, I would comment it to the original task.

Reply:
Sounds good to me